src/responses.py

```diff
@@ -9,31 +12,39 @@ def get_name(organization):
 
 
 class OrganizationCatalogResponse:
-    def __init__(self, organization, datasets, dataservices, concepts, informationmodels):
-        self.id = organization["organizationId"]
+    def __init__(self, organization: dict, datasets: ParsedContent, dataservices: ParsedContent,
+                 concepts: ParsedContent, informationmodels: ParsedContent):
+        if "organizationId" in organization.keys():
+            self.id = organization["organizationId"]
         self.organization = {
             "orgPath": organization["orgPath"],
             "name": get_name(organization),
         }
-        self.dataset_count = datasets["hits"]["total"]
-        self.dataservice_count = dataservices["total"]
-        self.informationmodel_count = informationmodels["page"]["totalElements"]
-        self.concept_count = concepts["page"]["totalElements"]
+        self.dataset_count = datasets.count if datasets else 0
+        self.dataservice_count = dataservices.count if dataservices else 0
+        self.informationmodel_count = informationmodels.count if informationmodels else 0
+        self.concept_count = concepts.count if concepts else 0
+
+    def has_content(self):
+        return self.dataservice_count + self.informationmodel_count + self.dataset_count + self.concept_count > 0
 
 
 class OrganizationCatalogListResponse:
     def __init__(self):
         self.org_list = []
 
     def add_organization_catalog(self, organization_catalog: OrganizationCatalogResponse):
-        print(organization_catalog)
-        self.org_list.append(organization_catalog.__dict__)
+        if organization_catalog.has_content():
+            self.org_list.append(organization_catalog.__dict__)
 
     def map_response(self):
         return {
             "organizations": self.org_list
         }
 
+    def count(self):
+        return self.org_list.__len__()
```

Comment:
don't use member methods that start with a double underscore (e.g. __len__ or __dict__), they're not meant to be called by developers. Instead, use global methods len() and dict()
```suggestion
        return len(self.org_list)
```